tools/cldr-apps/src/main/java/org/unicode/cldr/web/MailSender.java

```diff
@@ -41,7 +43,7 @@
  */
 public class MailSender implements Runnable {
 
-    private long waitTill = 0;
+    // private long waitTill = 0;
```

Comment:
minor suggestion for next time: delete instead of commenting out

Reply:
the implementation was already commented out, this just removes the warning